pandas/core/indexes/period.py

```diff
@@ -247,6 +247,10 @@ def _simple_new(cls, values, name=None, freq=None, **kwargs):
     # ------------------------------------------------------------------------
     # Data
 
+    @property
```

Comment:
shouldn't this be in the base class, e.g. all EA's do this by default?

Reply:
in any event you can write this as
``_eadata = _data``

Reply:
Same as the other PR; _eadata is a temporary shim